build-windows/build.sh

```diff
@@ -40,6 +40,17 @@ if [ "${CLASSICAL}" == "1" ]; then
   mkdir -p /root/out/x86_32/templates
   cp -rvp bin/* /root/out/x86_32/templates
   rm -rf bin
+
+  $SCONS platform=windows arch=arm64 $OPTIONS target=editor use_mingw=yes use_llvm=yes mingw_prefix=/root/llvm-mingw
```

Comment:
I'd suggest adding
```
OPTIONS_LLVM="use_llvm=yes mingw_prefix=/root/llvm-mingw"
```
next to `OPTIONS` and use it everywhere as `$OPTIONS $OPTIONS_LLVM`.

The `use_mingw=yes` is redundant I think (we don't use it for non-LLVM builds), but it doesn't hurt to be explicit, so I guess you could put it in `OPTIONS`.